doc/man1/openssl-dhparam.pod.in

```diff
@@ -50,14 +50,15 @@ See L<openssl-format-options(1)> for details.
 
 =item B<-in> I<filename>
 
-This specifies the input filename to read parameters from or standard input if
+This specifies the input file to read parameters from or standard input if
 this option is not specified.
 
 =item B<-out> I<filename>
 
-This specifies the output filename parameters to. Standard output is used
-if this option is not present. The output filename should B<not> be the same
-as the input filename.
+This specifies the output file to write parameters to.
+Standard output is used if this option is not present.
+The output filename can be the same as the input filename,
+which leads to replacing the file contents in situ.
```

Comment:
```suggestion
which leads to replacing the file contents in place.
Note that output I/O is not atomic, the output is truncated before it is written.
```

Reply:
Hmm, with this change the first sentence would contain 'replace' twice,
and the note can be generalised..
How about the following?
```
which leads to replacing the file contents.
Note that file I/O is not atomic. The output file is truncated and then written.
```

Reply:
Fine.

Reply:
Done for all apps touched.